src/ImageUploader.tsx

```diff
@@ -41,21 +45,24 @@ const ImageUploader: React.FC<ImageUploaderProps> = ({ onClose }) => {
     };
 
     return (
-        <div className='dialog-container'>
-            <p>Upload multiple images</p>
-            <input type='file' accept='image/*' multiple onChange={handleFileChange} />
-            <div>
-                {images.map((image, index) => (
-                    <img key={index} src={image.url} alt={image.alt} style={{ width: '100px', margin: '10px' }} />
-                ))}
+        <div className='Page'>
+            <div className='dialog-container'>
+                <p>Upload multiple images</p>
+                <input type='file' accept='image/*' multiple onChange={handleFileChange} style={{ display: 'none' }} />
+                <div>
+                    {images.map((image, index) => (
+                        <img key={index} src={image.url} alt={image.alt} style={{ width: '100px', margin: '10px' }} />
+                    ))}
+                </div>
+                {/*<p>Total Images: {images.length}</p>*/}
+                {<BoundBox />}
```

Comment:
Let's move the Bounding box component out of the image uploader and save it for a results page.